### PR TITLE
Temporarily disable creating custom embeddings if data is in GCS.

### PR DESCRIPTION
**Pull request by keyurva**
* Disabling it temporarily until we have full support for GCS data based embeddings.

### CODE REVIEW

**Review by shifucun — APPROVED** [review]
Add a todo to have an option in docker run to enable/disable NL functions?

File: server/routes/admin/html.py [hunk 1]
```diff
@@ -47,6 +47,9 @@ def load_data():
   output_dir = os.path.join(sql_data_path, 'data')
   nl_dir = os.path.join(output_dir, "nl")
   sentences_path = os.path.join(nl_dir, "sentences.csv")
+  # TODO: Support enabling / disabling NL via env / Dockerfile.
+  # TODO: Enable NL for GCS paths once we add support for it.
+  load_nl = not sql_data_path.lower().startswith("gs://")
```
Comment (pradh) [inline server/routes/admin/html.py:52]: Should we have a flag to explicitly enable/disable NL, even if sentences.csv is local?

Comment (keyurva) [reply server/routes/admin/html.py:52]: How about using `ENABLE_MODEL` for this?

Comment (pradh) [reply server/routes/admin/html.py:52]: Good idea

**Pull request merged**